server/test/src/main/antlr4/org/eclipse/usecase/UseCasePreprocessor.g4

```diff
@@ -78,15 +84,15 @@ constantUsage
    ;
 
 paragraphStatement
-   : (paragraphUsage | paragraphDefinition) diagnostic* STOP
+   : (paragraphUsage (INOF sectionUsage)?| paragraphDefinition) diagnostic* STOP
```

Comment:
```suggestion
   : (paragraphUsage (INOF sectionUsage STOP)?| paragraphDefinition) diagnostic* STOP
```
otherwise we are allowing something like `{$# PAR1 OF {$@ SEC1 | diag1}`